docs/DeveloperGuide.md

```diff
@@ -208,8 +208,31 @@ the filtered transaction with a success message.
     - 3a1. UniCa$h displays a message saying no results found.
     - Use Case resumes at step 1
 
+**Use Case: UC04 - Editing a Transaction**
 
-**Use Case: UC04 - Listing All Transactions**
+**MSS:**
+1. User enters the command to edit an existing transaction's fields in the correct format.
+2. User submits the request.
+3. UniCa$h makes the edits specified in the command and displays all the fields of that transaction.
+
+   Use Case ends.
+
+**Extensions**
+- 2a. User provides a field in an incorrect format. (e.g. amount is not expressed as a float, name is empty,
```

Comment:
See my message in the chat about why framing this as `2a` extension seems weird, would be good to standardize this